pyproject.toml

```diff
@@ -1,12 +1,115 @@
 [build-system]
 requires = [
-	"wheel", "setuptools",
+	"setuptools",
+	"wheel",
```

Comment:
Do you need the explicit wheel dep?

Reply:
Surprisingly enough, I do indeed use wheel, to mark my wheel as python ABI agnostic
The usage is with the line:
```python
from wheel.bdist_wheel import bdist_wheel as orig_bdist_wheel

...

class bdist_wheel(orig_bdist_wheel):
    ...
```